src/pages/questionPage.js

```diff
@@ -28,33 +28,58 @@ export const initQuestionPage = () => {
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
-  for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
-    const answerElement = createAnswerElement(key, answerText);
+  answersListElement.innerHTML = '';
+  const answers = currentQuestion.answers;
+
+  // Function to handle answer selection
+  const selectAnswer = (questionIndex, selectedOption, chosenItem) => {
```

Comment:
I think your `selectAnswer` function should be outside of this loop (my function and Ahmad's function is also outside as an independent function). I also noticed that `chosenItem` is expected as a param but you don't use it at all inside your function